src/main/docs/guide/introduction/upgrading.adoc

```diff
@@ -50,3 +78,71 @@ If the Netty based server implementation is being used, another dependency must
 dependency:io.micronaut.rxjava2:micronaut-rxjava2-http-server-netty[gradleScope="implementation"]
 
 NOTE: We recommend switching to Project Reactor as that is the implementation used internally by Micronaut. Adding a dependency to RxJava2 will result in both implementations in the runtime classpath of your application.
+
+=== Breaking Changes
+
+Review the section on <<breaks, Breaking Changes>> and update your affected application code.
+
+=== Automating Upgrades with OpenRewrite
+
+https://docs.openrewrite.org/[OpenRewrite^] is an automated refactoring tool that you can use to make many of the required upgrade changes. This works with Micronaut applications written in Java, but OpenRewrite doesn't currently support Kotlin or Groovy. Like any automated tool it does much of the work for you, but be sure to review the resulting changes and manually make any changes that aren't supported by OpenRewrite, for example converting from RxJava2 to Reactor.
+
+Adding OpenRewrite support to your build is easy, it just requires adding the Gradle or Maven plugin and configuring the plugin to use the Micronaut upgrade recipe.
+
+See the https://micronaut.io/launch?features=openrewrite&lang=JAVA&build=GRADLE&activity=diff[Gradle feature diff^] or the https://micronaut.io/launch?features=acme&lang=JAVA&build=MAVEN&activity=diff[Maven feature diff^] to see the required build script changes.
+
```

Comment:
The Maven link uses `acme` feature instead of `openrewrite`

Reply:
Nice catch!